test/t/test_property_value.cpp

```diff
@@ -373,3 +373,35 @@ TEST_CASE("create encoded property values from different bool types") {
     REQUIRE(b1.hash() == b2.hash());
 }
 
+TEST_CASE("property equality comparison operator") {
+    vtzero::data_view k{"key"};
```

Comment:
This is not a good use if a data_view. You are pointing to a string that you reference, that memory is not valid during the rest of the test.

Reply:
This is copied directly from here: https://github.com/mapbox/vtzero/blob/c59d9dd571ce9060bca704c429e60bd17c52ab09/test/t/test_property_value.cpp#L295-L303 - should I create a string explicitly before?